release.sh

```diff
@@ -27,126 +27,29 @@ check_git_branch() {
 
 check_git_branch
 
-# update to the latest
-git pull origin master
+sed -i -e "s/version = '.*';$/version = '$VERSION_NO_V';/" src/0_config.js
+sed -i -e "s/version = '.*';$/version = '$VERSION_NO_V';/" test/web-config.js
 
-read -p "Update CHANGELOG.md?" -n 1 -r
-echo
-if [[ $REPLY =~ ^[Yy]$ ]]
-then
-  vi CHANGELOG.md
-  git commit -am "Updated CHANGELOG.md"
-fi
-
-echo "Building files"
-
-read -p "Update 0_config.js? " -n 1 -r
-echo
-if [[ $REPLY =~ ^[Yy]$ ]]
-then
-  sed -i -e "s/version = '.*';$/version = '$VERSION_NO_V';/" src/0_config.js
-  sed -i -e "s/version = '.*';$/version = '$VERSION_NO_V';/" test/web-config.js
-fi
-
-read -p "Update package.json? " -n 1 -r
-echo
-if [[ $REPLY =~ ^[Yy]$ ]]
-then
-  sed -i -e "s/\"version\":.*$/\"version\": \"$VERSION_NO_V\",/" package.json
-  sed -i -e "s/\"build\":.*$/\"build\": \"$VERSION_NO_V\"/" package.json
-fi
-
-read -p "Bump CHANGELOG version? " -n 1 -r
-echo
-if [[ $REPLY =~ ^[Yy]$ ]]
-then
-  sed -i -e "s/## \[VERSION\] - unreleased/## [$VERSION] - $DATE/" CHANGELOG.md
-fi
-
-read -p "Update bower.json? " -n 1 -r
```

Comment:
this file seems not used anymore. I left it for now in case removing it blows some things up, but we no longer have an option to write to it